src/shadowbox/scripts/update_mmdb.sh

```diff
@@ -6,17 +6,17 @@
 # IP Geolocation by DB-IP (https://db-ip.com)
 
 TMPDIR="$(mktemp -d)"
-FILENAME="dbip-country-lite.mmdb"
+FILENAME="ip-country.mmdb"
 
 # We need to make sure that we grab an existing database at install-time
-for monthdelta in {0..11}; do
-    if [[ $monthdelta -gt 10 ]]; then
+for monthdelta in {0..10}; do
+    newdate=$(date --date="-$monthdelta month" +%Y-%m)
+    ADDRESS="https://download.db-ip.com/free/ip-country-${newdate}.mmdb.gz"
+    curl --fail --silent "${ADDRESS}" -o "$TMPDIR/$FILENAME.gz" > /dev/null && break
+    if (( $monthdelta == 10 )); then
```

Comment:
You can drop the $ here
`(( ))` interprets strings as variable names
(does the $ work?)

Reply:
It does, but fixed anyways